run_flax_speech_recognition_seq2seq_streaming_review.py

```diff
@@ -0,0 +1,1391 @@
+        "language": language_code,
+        "tags": ["audio", "asr", "automatic-speech-recognition", "hf-asr-leaderboard"],
+        "license": "apache-2.0",
+        "finetuned_from": model_args.model_name_or_path,
+        "tasks": ["asr"],
+        "dataset": data_args.dataset_name,
+        "dataset_args": {"name": data_args.dataset_config_name},
+        "source": "flax",
+        "eval_lines": [],
+        "eval_results": None,
+        "hyperparameters": {
+            "learning_rate": training_args.learning_rate,
+            "lr_scheduler_type": training_args.lr_scheduler_type,
+            "per_device_train_batch_size": training_args.per_device_train_batch_size,
+            "total_train_batch_size_per_node": train_batch_size // num_of_hosts,
+            "total_train_batch_size": train_batch_size,
+            "total_optimization_steps": f"{(data_args.num_train_steps - training_state['step']):,}",
+            "starting_optimization_step": f"{training_state['step']:,}" if training_state['step'] > 0 else None,
+            "finishing_optimization_step": f"{data_args.num_train_steps:,}",
+            "num_train_dataset_workers": f"{num_workers}",
+            "num_hosts": f"{num_of_hosts}",
+            "total_num_training_examples": f"{data_args.num_train_steps * train_batch_size:,}",
+            "steps_per_epoch": "_To be computed after first epoch_",
+            "num_beams": model_args.num_beams,
+        },
+        # TODO: Adapt https://github.com/huggingface/transformers/blob/main/src/transformers/modelcard.py#L855
+        # "hyperparameters": training_args.to_sanitized_dict()
+    }   
+        
+    if training_args.gradient_accumulation_steps > 1:
+        training_summary["hyperparameters"]["gradient_accumulation_steps"] = f"{training_args.gradient_accumulation_steps:,}"
+        training_summary["hyperparameters"]["effective_total_train_batch_size"] = f"{train_batch_size * training_args.gradient_accumulation_steps:,}"
+    
+    if model_args.dropout or model_args.attention_dropout or model_args.activation_dropout or model_args.encoder_dropout or model_args.decoder_dropout:
+        training_summary["hyperparameters"]["dropout"] = True
+        if model_args.dropout:
+            training_summary["hyperparameters"]["dropout_probability"] = model_args.dropout
+        if model_args.attention_dropout:
+            training_summary["hyperparameters"]["attention_dropout_probability"] = model_args.attention_dropout
+        if model_args.activation_dropout:
+            training_summary["hyperparameters"]["activation_dropout_probability"] = model_args.activation_dropout
+        if model_args.encoder_dropout:
+            training_summary["hyperparameters"]["encoder_dropout_probability"] = model_args.encoder_dropout
+        if model_args.decoder_dropout:
+            training_summary["hyperparameters"]["decoder_dropout_probability"] = model_args.decoder_dropout
+
+    # Create README if it does not exist
```

Comment:
Again here, we're really looking for a barebones Flax example (as minimalistic as possible really), so it's fine if we don't have a model card README

Reply:
One thing we found extremely useful when using the PyTorch `Trainer` was the automatic model card creation. This is of course your decision, but in our experience, there is no coming back, it really adds value to see an up to date model card every time there is a new checkpoint saved.